webpack.config.js

```diff
@@ -3,11 +3,10 @@ const ExtractTextPlugin = require("extract-text-webpack-plugin");
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
-  context: path.join(__dirname, 'src'),
+  context: path.join(__dirname, 'src/client/'),
```

Comment:
Using constants for paths and names will be also a good practice (defined in a separate config file)